package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@dojo/framework": "file:deps/dojo-framework-4.0.5.tgz",
+    "resize-observer-polyfill": "^1.5.0",
```

Comment:
With `dojo/framework@next` you shouldn't have to import the `resize-observer-polyfill`